src/z_dateformat.erl

```diff
@@ -304,14 +305,26 @@ tag_to_value($w, Date, _, _Options) ->
 tag_to_value($W, {Y,M,D}, _, _Options) ->
    integer_to_list(year_weeknum(Y,M,D));
 
+% Absolute year with at least 4 digits, padded with zeroes (e.g. '0003')
+tag_to_value($x, {Y, _, _}, _, _Options) when Y >= 1000; Y =< -1000 ->
+    integer_to_list(erlang:abs(Y));
+tag_to_value($x, {Y, _, _}, _, _Options) when Y < 0 ->
+    io_lib:format("~4..0B", [0 - Y]);
```

Comment:
What’s the advantage of this over `abs/1`?

Reply:
Simpler :-)